skymp5-server/cpp/addon/ScampServer.cpp

```diff
@@ -150,6 +150,16 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
 
     auto serverSettings = nlohmann::json::parse(buffer.str());
 
+    if (serverSettings.find("weaponStaminaModifiers") !=
+        serverSettings.end()) {
+      if (serverSettings.at("weaponStaminaModifiers").is_object()) {
+        auto modifiers = serverSettings.at("weaponStaminaModifiers")
+                           .get<std::unordered_map<std::string_view, float>>();
```

Comment:
Please use `std::string` instead of `std::string_view`.

as far as I know `std::string_view` is a non-owning object. we can't make it from temporary objects.

I'm not 100% sure but let's not experiment

Reply:
yep, sure, `std::string_view` is dangling here, I'll fix this today

Reply:
ok